keps/NEXT_KEP_NUMBER

```diff
@@ -1 +1 @@
-34
+35
```

Comment:
drop this (will hit conflicts and is in the process of being removed from the template)

Reply:
Do you know what the process is now? the kep no is in the filename, which is pretty awkward (have to rename the file multiple times if conflict)

Reply:
leave the filename as-is for now. can rename to 0000 or drop the prefix or adjust to whatever the replacement scheme becomes.

Reply:
Use a date. See https://github.com/kubernetes/enhancements/pull/703